linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/var_smartcard_drivers.var

```diff
@@ -0,0 +1,57 @@
+documentation_complete: true
+
+title: 'OpenSC Smart Card Drivers'
+
+description: |-
+    Choose the Smart Card Driver in use by your organization.
+    <br />For DoD, choose the <tt>cac</tt> driver.
+    <br />If your driver is not listed and you don't want to use the
+    <tt>default</tt> driver, use the <tt>other</tt> option and
+    manually specify your driver.
+
+type: string
+
+operator: equals
+
+interactive: true
+
+options:
+    default: default
+    acos5: acos5
+    akis: akis
+    asepcos: asepcos
+    atrust-acos: atrust-acos
+    authentic: authentic
+    belpic: belpic
+    cac: cac
+    cardos: cardos
+    coolkey: coolkey
+    cyberflex: cyberflex
+    dnie: dnie
+    entersafe: entersafe
+    epass2003: epass2003
+    flex: flex
+    gemsafeV1: gemsafeV1
+    gids: gids
+    gpk: gpk
+    iasecc: iasecc
+    incrypto34: incrypto34
+    isoApplet: isoApplet
+    itacns: itacns
+    jpki: jpki
+    MaskTech: MaskTech
+    mcrd: mcrd
+    muscle: muscle
+    myeid: myeid
+    npa: npa
+    oberthur: oberthur
+    openpgp: openpgp
+    other: 
```

Comment:
Is this empty value intentional? It will get translated to "None" by the build system.

Reply:
Yes, it is, so that it is changeable.